tests/test_usgs_stac.py

```diff
@@ -0,0 +1,40 @@
+import os
+import unittest
+from stactools.landsat.usgs_stac import convert_usgs_stac
+
+
+class UsgsStacTest(unittest.TestCase):
+
+    def test_tm(self):
+        mtl_xml_href = "tests/data-files/usgs-stac/tm/LT04_L2SP_002026_19830110_20200918_02_T1_MTL.xml"  # noqa
+
+        item = convert_usgs_stac(mtl_xml_href)
+        item.validate()
+
+        destination = "examples/usgs-stac-convert/tm"
```

Comment:
I don't think we should create the `examples` during unit test execution, because that would update e.g. the `created` attribute every test run. A `scripts/create_examples` would be more appropriate. Here in the unit tests, use a `TemporaryDirectory`.

Reply:
Yep. No longer using tests to create the examples.